website/static/js/comment.js

```diff
@@ -14,7 +14,44 @@ require('jquery-autosize');
 var osfHelpers = require('js/osfHelpers');
 var CommentPane = require('js/commentpane');
 var markdown = require('js/markdown');
+var atjsConfig = require('js/atjsConfig');
 
+var input = $('.atwho-input');
+var nodeId = window.nodeId;
+
+var getContributorList = function(input, nodeId) {
+    var url = osfHelpers.apiV2Url('nodes/' + nodeId + '/contributors/', {});
+    var request = osfHelpers.ajaxJSON(
+        'GET',
+        url,
+        {'isCors': true});
+    request.done(function(response) {
+        var activeContributors = response.data.filter(function(item) {
+            return item.embeds.users.data.attributes.active === true;
+        });
+        var data = activeContributors.map(function(item) {
+            return {
+                'id': item.id,
+                'name': item.embeds.users.data.attributes.given_name,
+                'fullName': item.embeds.users.data.attributes.full_name,
+                'link': item.embeds.users.data.links.html
+            };
+        });
+        // for any input areas that currently exist on page
+        input.atwho('load','@', data).atwho('load', '+', data).atwho('run');
+        // for future input areas so that data doesn't need to be reloaded
+        atjsConfig.atConfig.data = data;
```

Comment:
Minor: It would be nice to avoid mutating this object that we're importing (`atConfig`)

```js
// this would go whereever you initialize At.js
getContributorList().then(function(contributors) {
    var config = $.extend({}, atjsConfig.atConfig, {data: contributors});
    $.atjs(config);
};
```

Not a blocking refactor, though.